docs/components/backdrop.md

```diff
@@ -0,0 +1,48 @@
+# Backdrop
+
+[component-header:sl-backdrop]
+
+The backdrop signals to the user of a state change within the application and can be used for creating loaders, dialogs and more. In its simplest form, the backdrop component will add a dimmed layer over your application.
+
+```html preview
```

Comment:
Not sure why, but it seems this first example does no longer work, when activating the dark mode.

Tested in the preview-verxel link https://shoelace-git-backdrop-overlay.shoelace.vercel.app/components/backdrop

Repro
- open above link
- activate dark mode (top right corner button)
- open the backdrop
- try to close the backdrop -> not closable via click on the background